### PR TITLE
Add requirements to sam_pileup.py, fix script

### PR DESCRIPTION
**Pull request by mvdbeek**

### CODE REVIEW

File: tools/sam_pileup/sam_pileup.py [hunk 1]
```diff
@@ -21,50 +21,55 @@
...
+import tempfile
 from bx.cookbook import doc_optparse
 
-def stop_err( msg ):
-    sys.stderr.write( '%s\n' % msg )
-    sys.exit()
+
+def stop_err(msg, e=None):
+    if e is not None:
+        raise e
```
Comment (nsoranzo) [inline tools/sam_pileup/sam_pileup.py]: This way `msg` is never printed.

File: tools/sam_pileup/.shed.yml [hunk 2]
```diff
@@ -9,5 +9,6 @@ long_description: |
   dataset with consensus is generated.
 name: sam_pileup
 owner: devteam
+homepage_url: https://github.com/galaxyproject/tools-devteam/tree/master/tools/sam_pileup
```
Comment (nsoranzo) [inline tools/sam_pileup/.shed.yml]:
```suggestion
homepage_url: https://www.htslib.org/
```

**Comment by nsoranzo** [comment]
After spending sometime cleaning up the overlong Python code, I realised this doesn't need to use a Python script at all... I'll work on that later or tomorrow.

**Comment by mvdbeek** [comment]
It's probably not worth investing too much time though, noone should be using pileup format in 2020 anyway.

**Comment by nsoranzo** [comment]
@mvdbeek I got rid of the Python, can you double check the changes?

**Comment by mvdbeek** [comment]
Looks good, thanks @nsoranzo !

**Pull request merged**